3.8/debian-10/rootfs/opt/bitnami/scripts/librabbitmq.sh

```diff
@@ -62,13 +62,23 @@ rabbitmq_validate() {
         fi
     }
 
+    check_fqdn() {
+        if [[ "${!1}" == *.* ]]; then
+            if [[ "${RABBITMQ_USE_LONGNAME}" = true ]] ; then
```

Comment:
I misinterpreted the condition and used `= true` in my example. Right now, this would actually throw an error when you are using `RABBITMQ_USE_LONGNAME` correctly. My bad! This should have been:

```suggestion
            if [[ "${RABBITMQ_USE_LONGNAME}" = false ]] ; then
```

Reply:
And I did not catch it, thank you :-)